src/Equinox/Core.fs

```diff
@@ -13,7 +13,7 @@ type IStream<'event, 'state> =
     abstract LoadEmpty: unit -> struct (StreamToken * 'state)
 
     /// Obtain the state from the target stream
-    abstract Load: allowStale: bool * requireLeader: bool * ct: CancellationToken -> Task<struct (StreamToken * 'state)>
+    abstract Load: maxStaleness: TimeSpan * requireLeader: bool * ct: CancellationToken -> Task<struct (StreamToken * 'state)>
```

Comment:
Suggestion: `maxStaleness` -> `maxAge`

Reply:
Good call, done (still using that as the name for the mode, but #386 already calls that out)